packages/api/tests/app/test-missing-dynamo-env-var.js

```diff
@@ -0,0 +1,14 @@
+const test = require('ava');
+
+const { MissingRequiredEnvVarError } = require('@cumulus/errors');
+
+process.env.INIT_ENV_VARS_FUNCTION_TEST = 'true';
+
+test.serial('index throws error if environment variable for Dynamo tables parameter name is missing', async (t) => {
```

Comment:
Does this need to be serial if it's the only test?

Reply:
Excellent point 😅